Resources/Prototypes/SimpleStation14/Entities/Mobs/Customization/ears.yml

```
@@ -93,7 +93,7 @@
   id: EarsShadekin
   bodyPart: HeadTop
   markingCategory: HeadTop
-  speciesRestriction: [SlimePerson, Reptilian]
+  speciesRestriction: [SlimePerson, Reptilian, Shadekin]
```

```suggestion
  speciesRestriction: [Shadekin]
```
Now that Shadowking are a thing, we probably don't need Reps stepping on their turf. For Slimes I understand, but we probably want Shadowkin looking distinct, since the ears will be a huge identifier of the race, whereas for Reps it's the snout, colours, and tail, and for Slimes it's the general transparency.